Src/Core/Specialized/AsyncFunctionAssertions.cs

```diff
@@ -72,14 +72,14 @@ public void ShouldNotThrow(string because = "", params object[] becauseArgs)
                 Task task = Subject();
                 task.Wait();
```

Comment:
Side note: Maybe look into [Task.WaitAndUnwrapException() extension ](https://gist.github.com/tomachristian/83cbf6618cdcd60949e90787347b9c36), it keeps the code cleaner IMO and you don't have too look into InnerException.